diktat-rules/src/test/resources/test/smoke/save.toml

```diff
@@ -0,0 +1,10 @@
+[general]
+execCmd="./ktlint -R diktat-1.0.0-rc.2.jar"
+tags = ["smokeTest"]
+description = "SmokeTest"
+suiteName = "SmokeTest"
+
+[fix]
```

Comment:
Shouldn't it be `fix-and-warn`? We have some unfixed warnings in smoke tests

Reply:
Fixed